com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/MyLanguageServerImpl.java

```diff
@@ -162,10 +106,23 @@ void retrieveAndStoreConfiguration() {
         new WorkspaceServerCapabilities(workspaceFoldersOptions);
     capabilities.setWorkspace(workspaceServiceCapabilities);
 
-    copybookService.setWorkspaceFolders(params.getWorkspaceFolders());
     return supplyAsync(() -> new InitializeResult(capabilities));
   }
 
+  private void addLocalFilesWatcher() {
+    settingsService
+        .getConfiguration(LOCAL_PATHS.label)
+        .thenAccept(it -> watchingService.addWatchers(toStrings(it)));
+  }
+
+  private List<String> toStrings(List<Object> it) {
```

Comment:
The same as `CobolWorkspaceServiceImpl#toString`. I think that it could be a method of `SettingsService`.